esbuild/plugins.ts

```diff
@@ -67,6 +67,10 @@ export const getPlugins = ({
           from: toPosix(path.join(SRC_DIR, 'assets', '**', '*')),
           to: toPosix(path.join(outDir, 'assets')),
         },
+        {
+          from: ['./node_modules/@fontsource/titillium-web/files/*'],
```

Comment:
Might need to update copy plugin to only copy required files (maybe add a filter function).

Reply:
`@fontsource/titillium-web` ships all font variants in the `files` directory. We should only copy the variants that we are using. This way we are not going to increase the extension bundle size with assets that are not used at all.